test/Handler/FlagMentorSpec.hs

```diff
@@ -0,0 +1,65 @@
+module Handler.FlagMentorSpec (spec) where
+
+import TestImport
+
+import Database.Persist.Sql (toSqlKey)
+import Utils.Flag
+
+spec :: Spec
+spec = withApp $ do
+
+    describe "getFlagMentorR" $ do
+        it "asserts flag link doesn't show to own user" $ do
+            userEntity <- createUser "foo"
+            authenticateAs userEntity
+
+            get ProfileR
+            htmlCount ".flag-wrapper" 0
+
+        it "asserts flag link doesn't show to anonymous users" $ do
+            userEntity <- createUser "foo"
+            let (Entity _ user) = userEntity
+
+            get $ UserR (userIdent user)
+            htmlCount ".flag-wrapper" 0
+
+        -- it "asserts route access for valid arguments" $ do
+        --     userEntity <- createUser "foo"
+        --     authenticateAs userEntity
+        --
+        --     userEntity' <- createUser "bar"
+        --     let (Entity flaggedId _) = userEntity'
+        --
+        --     token <- getFlagTokenFromCsrf flaggedId Flag
```

Comment:
This causes an error, so I had to comment it out. Seems that return value of `Handler Text` doesn't work well inside the test. I wonder how to overcome it?